### PR TITLE
[docs] Update support details in CSV 

### PR DESCRIPTION
**Pull request by sebsoto**
This commit directs customers to use the Red Hat Customer Portal instead
of bugzilla to open issues with the operator.

### CODE REVIEW

**Comment by aravindhp** [comment]
/cc @codyhoag @Anandnatraj 

Please review.

**Review by codyhoag — APPROVED** [review]
LGTM

**Comment by reestr** [comment]
@aravindhp - troubleshooting guide should be in the product documentation, we shouldn't be pointing entitled customers to github.  Can we update that link to product documentation and then publish?

**Comment by aravindhp** [comment]
> @aravindhp - troubleshooting guide should be in the product documentation, we shouldn't be pointing entitled customers to github. Can we update that link to product documentation and then publish?

Yes we can do that. @codyhoag is there a ghost link that @sebsoto can use?

File: deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -127,9 +132,9 @@ spec:
     - [AWS](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-aws.md)
 
     ### Reporting issues
-    Issues with this distribution of WMCO can be opened against the [Windows Container Bugzilla component](https://bugzilla.redhat.com/buglist.cgi?product=OpenShift%20Container%20Platform&component=Windows%20Containers&resolution=---).
+    Support for this distribution of WMCO requires a subscription. Support should be requested through the Red Hat Customer Portal.
     Please read through the [troubleshooting document](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/TROUBLESHOOTING.md)
```
Comment (aravindhp) [inline deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml:136]: Troubleshooting link needs to be replaced with a link to OpenShift docs.

File: deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -26,7 +26,12 @@ spec:
     container runtime installed. The operator completes all the necessary steps to configure the underlying VM so that it can join
     the cluster as a worker node.
 
+    Usage of this operator requires a subscription. Users looking to deploy Windows containers workloads in production
+    clusters should acquire a subscription before attempting to install this operator. Users without a subscription can
+    try the community WMCO, a distribution which lacks official support.
```
Comment (aravindhp) [inline deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml:31]: `community operator`

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Comment by codyhoag** [comment]
@aravindhp @sebsoto ghost link for Troubleshooting guide in official docs: https://docs.openshift.com/container-platform/4.6/support/troubleshooting/troubleshooting-windows-container-workload-issues.html

File: deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (codyhoag) [reply deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml:136]: https://docs.openshift.com/container-platform/4.6/support/troubleshooting/troubleshooting-windows-container-workload-issues.html

**Review by aravindhp — CHANGES_REQUESTED** [review]
@sebsoto please change the maintainer [name](https://github.com/openshift/windows-machine-config-operator/blob/master/deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml#L348) to `Red Hat, Windows Container Support for OpenShift`

cc: @Anandnatraj

**Review by aravindhp — APPROVED** [review]
/approve

**Comment by Anandnatraj** [comment]
LGTM

**Pull request merged**